drivers/bluetooth/hci/vendor/infineon/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library_sources(cyw43xxx.c)
```

Comment:
I don't think this is necessary. You can just use that directly in the parent CMakeLists.txt as the other drivers do.